apps/web/components/ui/BlockActions/BlockActions.vue

```diff
@@ -61,8 +61,11 @@ const props = defineProps<{ index: number; isLastBlock: boolean }>();
 
 const emit = defineEmits(['edit', 'delete', 'change-position']);
 
+const { openDrawerWithView } = useSiteConfiguration();
+
 const triggerEdit = () => {
   emit('edit', props.index);
+  openDrawerWithView('blocksSettings', 'right');
```

Comment:
Please use the feature flag here until development on all settings is finished.

```suggestion
if (useRuntimeConfig().public.experimentalBlockEditForm) {
  openDrawerWithView('blocksSettings', 'right');
}
```